packages/editor/src/components/post-publish-panel/postpublish.js

```diff
@@ -44,7 +44,7 @@ const getFuturePostUrl = ( post ) => {
 function CopyButton( { text, onCopy, children } ) {
 	const ref = useCopyToClipboard( text, onCopy );
 	return (
-		<Button variant="secondary" ref={ ref }>
+		<Button __next40pxDefaultSize variant="secondary" ref={ ref }>
```

Comment:
The thing that appears after you just publish a post.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/a8900b38-16a2-40e7-bcf3-02740dce0517" alt="PostPublishPanel, before" width="273">|<img src="https://github.com/user-attachments/assets/10797f8f-7562-49dc-8acf-5fe5e71e1cdb" alt="PostPublishPanel, after" width="273">|